.github/workflows/pr.yml

```
@@ -1,30 +1,28 @@
 # This GitHub Actions workflow automates the process of
 # publishing dataset collections to a staging environment
-# and creating a pull request (PR) in the veda-config repository
-# with the dataset configuration.
 # It is triggered by a pull request to the main branch
 # that modifies any files within the ingestion-data/dataset-config/ directory
 # The workflow includes steps to
 #   - publish the datasets,
-#   - create a PR in veda-config repository,
 #   - constantly updates the status of the workflow in the PR comment
 
-name: Publish collection to staging and create dataset config PR
+name: Publish collection to staging
 
 on:
   pull_request:
-    branches:
-      - main
+    branches: ['main', 'feat/automated-staging-publish']
```

```suggestion
    branches: ['main']
```
I can remove this later but I added it to prove that it can be triggered on pushes to a branch